EventFilter/L1TRawToDigi/plugins/implementations_stage2/MPUnpacker_0x10010033.cc

```diff
@@ -2,24 +2,13 @@
 
 #include "FWCore/Framework/interface/MakerMacros.h"
 
-#include "EventFilter/L1TRawToDigi/interface/Unpacker.h"
+#include "EventFilter/L1TRawToDigi/plugins/UnpackerFactory.h"
```

Comment:
maybe its possible to break apart UnpackerFactory to avoid this direct inclusion in the future? Seems a duplicate plugin definition will eventually happen here (not urgent)